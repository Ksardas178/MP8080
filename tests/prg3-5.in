LXI D b00001111 4	//00000100|00001111 = 1039
LXI B b00010000 3	//00000011|00010000 = 784
	
MOV A E
SUB C
MOV E A

MOV A D
SBB B
MOV D A

XCHG //DE <-> HL
SHLD o200 o000
HLT

//255


