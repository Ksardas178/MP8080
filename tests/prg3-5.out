
[32;1mInput is correct
[0m
[30;1mCode analysis results:
[0m______________________________
00000000: 00010001
00000001: 00001111
00000010: 00000100
00000011: 00000001
00000100: 00010000
00000101: 00000011
00000110: 01111011
00000111: 10010001
00001000: 01011111
00001001: 01111010
00001010: 10011000
00001011: 01010111
00001100: 11101011
00001101: 00100010
00001110: 10000000
00001111: 00000000
00010000: 01110110
